.github/workflows/build-test.yml

```diff
@@ -670,17 +670,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-
-    # We don't expect to keep being clippy-clean on nightly Rust,
-    # so the "toolchain boilerplate" is deliberately omitted
-    # This just means we end up testing on the pinned stable even in the cronjob
-    # CI, which isn't that useful but specially omitting that task is more work than it's worth
-
-    # Clippy job > Install and run clippy steps
-    - name: Install clippy
-      run: rustup component add clippy
-
-    - uses: actions-rs/clippy-check@v1.0.7
+    - uses: actions-rs/toolchain@v1
+      with:
+          toolchain: stable
```

Comment:
Please hardcode 1.67, otherwise a Rust release can break our CI.

Reply:
@Manishearth can I hardcode `1.67` here or would I have to hardcode `1.67.0`?